src/relay/transforms/simplify_expr.cc

```diff
@@ -326,6 +326,56 @@ static std::vector<int> GetTransposeAxisOrder(const Call& call, int ndim) {
   return std::move(attr_axes);
 }
 
+/*!
+ * \brief SimplifyClipCastClip matches the pattern clip->cast->clip and fuse clips based on Clip
+ *    min/max values.
+ *
+ * Example:
+ *   %1 = clip(%0, a_min=0f, a_max=255f) [type=int32]
+ *   %2 = cast(%1, dtype="uint8") [type=uint8]
+ *   %3 = clip(%2, a_min=20f, a_max=66f) [type=uint8]
+ *
+ * Optimized to (fuse Clips):
+ *   %1 = clip(%0, a_min=20f, a_max=66f) [type=uint8]
+ *   %2 = cast(%1, dtype="uint8") [type=uint8]
+ */
```

Comment:
Thanks for the PR! @youxiudeshouyeren. This optimization behavior seems not thorough enough; It can be further optimized to  `%1 = clip(%0, a_min=20f, a_max=66f) [type=uint8]`.

Reply:
I realize it was my example that was written incorrectly, the first data type is int32. It should be:
```
 * Optimized to (fuse Clips):
 *   %1 = clip(%0, a_min=20f, a_max=66f) [type=int32]
 *   %2 = cast(%1, dtype="uint8") [type=uint8]
 */
```
I'll correct the example in the comment.